source/manual/restore-from-offsite-backups.html.md

```diff
@@ -27,6 +27,16 @@ on a Vagrant VM.
 
 On a fresh VM, you may require the following packages for this exercise:
 
+> You can use either your dev VM or if you have the space you can create a new new mysql server VM using the following command:
```

Comment:
`new` repeated twice `new new`

I recommend installing the pre-commit hooks that would flag these errors. Docs in the README :)

Reply:
well it would be a `new new` instance... I will have a look at the docs for the pre-commit hook. :)